src/locales/en.json

```
@@ -22,6 +22,19 @@
     "Empty": {
       "welcome": "Welcome on Cozy Notes",
       "after_created": "Once your Note is created, you can find it in your "
+    },
+    "SavingIndicator": {
+      "saved_just_now": "Saved just now.",
+      "saved_secs_ago": "Saved %{time} sec ago.",
+      "saved_min_ago": "Saved %{time} min ago.",
+      "saved_mins_ago": "Saved %{time} min ago.",
+      "saved_hour_ago": "Saved %{time} hour ago.",
+      "saved_hours_ago": "Saved %{time} hours ago.",
+      "saved_day_ago": "Saved %{time} day ago.",
+      "saved_days_ago": "Saved %{time} days ago.",
+      "saving": "Saving...",
```

```suggestion
      "saving": "Saving…",
```
(sorry, typography nerding, feel free to ignore these 😅)

---

I did hesitate, but these are more loading signs that suspension points, and people may not recognize them as such  with suspension points.

---

Yeeah fair enough 😀works for me